krun.py

```diff
@@ -330,8 +331,4 @@ def attach_log_file(config, resume):
     debug("arguments: %s"  % " ".join(sys.argv[1:]))
     parser = create_arg_parser()
     setup_logging(parser)
-
-    try:
-        main(parser)
-    except:
-        sys.exit(1)
+    main(parser)
```

Comment:
This is the top-level. The problem was, if an exception occurred (say) one line before this line, krun would exit silently. And this caused me some pain.